webapp/package.json

```diff
@@ -33,5 +34,10 @@
     "qs": "6.9.4",
     "Select2": "github:select2/select2#3.5.4",
     "tablesorter": "2.31.3"
+  },
+  "scripts": {
+    "build": "rm -rf static/* && npm run copy:images && node esbuild && npm run build:collectstatic",
+    "build:collectstatic": "TOOL_ENV=True ./manage.py collectstatic --noinput -i crashstats/*.css -i api/*.css -i documentation/*.css -i profile/*.css -i signature/*.css -i status/*.css -i supersearch/*.css -i tokens/*.css -i topcrashers/*.css",
```

Comment:
This is duplicated in `bin/test.sh`. Either `bin/test.sh` should call `npm run build:collectstatic` or we should pull the two calls into a shell script and call that. It's sufficiently complex that we want to centralize it into one place.

Reply:
fixed via https://github.com/mozilla-services/socorro/pull/6864/commits/2bb3372cff60ac8d9a052f8ae503138ba6453c9e

Reply:
Given how long and unwieldy this line is, I think I would have moved this to a shell script and called the shell script from both `bin/test.sh` and here. That gives us syntax highlighting, shellcheck, and better ergonomics for maintaining it.

But this is fine for now.

Reply:
ok, I'll keep an eye on it when we add JS to ESBuild's responsibility.